saleor/graphql/shop/filters.py

```diff
@@ -0,0 +1,10 @@
+import graphene
+
+
+class CountryFilterInput(graphene.InputObjectType):
+    in_shipping_zones = graphene.Boolean(
```

Comment:
Wouldn't be better to have something like: `attached_to_shipping_zones`? `in_shipping_zones` for first look sugested me that I can provide IDs of shipping zones. 
Just want to know what do you think

Reply:
now that you mention it I can see that it might be confusing. I'll change it.